rest-api-spec/src/main/resources/rest-api-spec/api/cat.cluster_manager.json

```diff
@@ -0,0 +1,51 @@
+{
+  "cat.cluster_manager":{
+    "documentation":{
+      "url":"https://opensearch.org/docs/latest/opensearch/rest-api/cat/cat-master/",
+      "description":"Returns information about the cluster manager node."
+    },
+    "stability":"stable",
+    "url":{
+      "paths":[
+        {
+          "path":"/_cat/cluster_manager",
+          "methods":[
+            "GET"
+          ]
+        }
+      ]
+    },
+    "params":{
+      "format":{
+        "type":"string",
+        "description":"a short version of the Accept header, e.g. json, yaml"
+      },
+      "local":{
+        "type":"boolean",
+        "description":"Return local information, do not retrieve the state from master node (default: false)"
+      },
+      "master_timeout":{
```

Comment:
Shouldn't this parameter name be changed to be inclusive?

Reply:
Hmmm, good question. I didn't think of it at that time. Since the deprecation of the parameter "master_timeout" will be handled in separate PR, maybe it is fair to keep it for now. 🙂

Reply:
I don't think we should introduce a new API with old parameter names. If a client moves to the `_cat/cluster_manager` API with this in place, they will continue to use the `master_timeout` parameter name. Then, when we deprecate the parameter, they will need to make another change to prevent breakage.

I would rather we use inclusive naming from the get-go in net-new APIs and code paths.

Reply:
I got it. Make sense not to introduce a new API with out-dated parameter name.
I realized that a new json file of rest-api-spec may not necessary, because the main code change I made is to give an alternative API path to `_cat/master`, but not adding a new API.
I will modify the code not to create a new json file of rest-api-spec, and find out a better way to deal with the API path change.
If that works, I will also modify the PR title and description, because the code change is not actually adding a new REST API.

Reply:
I modified the code to:
- Rename the current JSON rest-api-spec file `cat.master.json` to `cat.cluster_manager.json`
- Add path `_cat/cluster_manager` and  `_cat/master` path as the deprecated path
- Test both old and new path in one YAML file to test the both REST API path

Reply:
👍 please add an issue so we don't miss out on it

Reply:
Sure, I will put the issue link here, and also attach to #1549 . Thanks